UI/DOM.js

```diff
@@ -0,0 +1,115 @@
+class View {
+  constructor(model, isLogged) {
+    this.model = model;
+    this.isLogged = isLogged;
```

Comment:
No manipulation methods for isLogged state, so have no good way to change login/logout throw View.